ckan/migration/versions/093_7f70d7d15445_remove_activity_revision_id.py

```diff
@@ -51,9 +51,6 @@ def downgrade():
     op.add_column(u'system_info',
                   sa.Column(u'revision_id', sa.TEXT(), autoincrement=False,
                             nullable=True))
-    op.create_foreign_key(u'resource_view_resource_id_fkey', u'resource_view',
-                          u'resource', ['resource_id'], ['id'],
-                          onupdate=u'CASCADE', ondelete=u'CASCADE')
```

Comment:
no longer needed?

Reply:
I mentioned it above, this key exists when I'm trying to downgrade the DB version, so this line just prevents me from downgrading. And reverse function(upgrade) does not drop this index, so this line probably was added by mistake